ap-maintenance-wg-charter.html

```diff
@@ -0,0 +1,529 @@
+        <h2>Scope</h2>
+
+        <p>
+          The Working Group will maintain the <a href="https://www.w3.org/TR/activitypub/">ActivityPub</a> protocol,
+          as well as the <a href="https://www.w3.org/TR/activitystreams-core/">Activity Streams</a> and
+          <a href="https://www.w3.org/TR/activitystreams-vocabulary/">Activity Vocabulary</a> data models on
+          which it depends, as well as related Notes.
+
+        <section id="section-out-of-scope">
+          <h3 id="out-of-scope">Out of Scope</h3>
+          <p>
+            Changes that add new functionality
+            (<a href="https://www.w3.org/policies/process/#class-4">class 4</a>)
+            are out of scope.
+          </p>
+        </section>
+
+      </section>
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <p>Updated document status is available on the
+            <a href="https://www.w3.org/groups/wg/ap/publications/">group publication status page</a>.
+        </p>
+
+        <section id="normative">
+          <h3>
+            Normative Specifications
+          </h3>
+          <p>
+            The Working Group will deliver the following W3C normative specifications:
+          </p>
+          <dl>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitypub/">ActivityPub</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+                <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+                <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitystreams-core/">Activity Streams</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+              <p><b>Exclusion Draft:</b>  @@to be filled later@@
+              Associated @@to be filled later@@
+              <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitystreams-vocabulary/">Activity Vocabulary</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+                <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+                <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+
+            </dd>
+          </dl>
+        </section>
+      </section>
+
+	<section id="success-criteria">
+	  <h2>Success Criteria</h2>
+
+    <p>
+      In order to update the Recommendation, each substantive change is expected
+      to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least
+      two independent implementations</a> of every feature defined in the
+      specification, where interoperability can be verified by passing open test suites.
```

Comment:
```suggestion
      specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperating with each other.
```

Reply:
+1, merging this suggestion. (two+ implementations is a common requirement for w3c groups and beyond)